Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as

```diff
@@ -570,28 +572,20 @@ package com.flashlight.vnc
 					case Keyboard.F10  		: keysym = 0xFFC7; break;
 					case Keyboard.F11  		: keysym = 0xFFC8; break;
 					case Keyboard.F12  		: keysym = 0xFFC9; break;
+					case 91               : keysym = 0xFFEB; break; // Windows key
 					case Keyboard.SHIFT 	: 
 						keysym = 0xFFE1; 
 						shiftKeyDown = event.type == flash.events.KeyboardEvent.KEY_DOWN;
 						break;
 					case Keyboard.CONTROL	: keysym = 0xFFE3; break;
-					default: {
-						if (event.type == flash.events.KeyboardEvent.KEY_DOWN && event.ctrlKey) {
-							expectKeyInput = true;
-							return;
-						}
-						if (event.type == flash.events.KeyboardEvent.KEY_UP && expectKeyInput) {
-							expectKeyInput = false;
-							rfbWriter.writeKeyEvent(true,event.charCode);
-							rfbWriter.writeKeyEvent(false,event.charCode);
-							return;
-						}
-					}
 				}
 				
 				//logger.info("keysym "+keysym);
 				
 				rfbWriter.writeKeyEvent(event.type == flash.events.KeyboardEvent.KEY_DOWN,keysym);
+
+				// prevent onTextInput() from firing
+				//event.stopImmediatePropagation();
```

Comment:
It's a note to future-me to remind me that you can stop events propagating, I cargo culted it from this related [PR](https://github.com/rainforestapp/Flashlight-vnc/pull/9/files#diff-5e601c60176c463882f8ef03ad5d3f5cL542).

Enabling/disabling it doesn't seem to do anything though... So the comment is wrong.